docs/integration/KafkaIntegration.md

```diff
@@ -44,4 +44,136 @@ In the Designer the user can choose, for each source and sink, which schema vers
 At runtime Nussknacker determines the schema version of a message value and key in the following way:
 1. it checks `value.schemaId` and `key.schemaId` headers;
 2. if there are no such headers, it looks for a magic byte and schema version in the message, [in a format used by Confluent](https://docs.confluent.io/platform/current/schema-registry/serdes-develop/index.html#wire-format);
-3. if the magic byte is not found, it assumes the schema version chosen by the user in the scenario.
\ No newline at end of file
+3. if the magic byte is not found, it assumes the schema version chosen by the user in the scenario.
+
+  
+# Configuration
```

Comment:
A major question and difficulty I always had with configuration was trying to figure out where exactly in this tree of configuration settings the given setting should be? What is convention used in this documentation to answer this question? 

The answer is _I do not know, but may be I omitted something_. So, if this convention is not obvious, I would consider addition something like this:
_Unless stated otherwise all settings here are relative scenarioTypes.ScenarioTypeName.modelConfig key. You can find the high level structure of the configuration file [here](https://nussknacker.io/documentation/docs/installation_configuration_guide/#configuration-areas).

Reply:
:+1: